pyanaconda/modules/storage/partitioning/manual/manual_module.py

```diff
@@ -163,7 +167,15 @@ def _find_request_for_device(self, device, requests):
         :return: an instance of MountPointRequest or None
         """
         for request in requests:
-            if device is self.storage.devicetree.get_device_by_device_id(request.device_spec):
+            if request.device_spec:
+                rdevice = self.storage.devicetree.get_device_by_device_id(request.device_spec)
+            else:
+                rdevice = self.storage.devicetree.resolve_device(request.ks_spec)
+            if device is rdevice:
+                if not request.device_spec:
+                    request.device_spec = device.device_id
+                if not request.ks_spec:
+                    request.ks_spec = device.name
```

Comment:
I wonder if we should set the ks_spec value also in places other then parsing kickstart (like here), would it be valuable (and feasible) to strictly keep the value from ks there?

Reply:
Makes sense, fixed.